solr/solr-ref-guide/modules/query-guide/pages/common-query-parameters.adoc

```diff
@@ -400,6 +400,16 @@ If early termination is used, a `segmentTerminatedEarly` header will be included
 
 Similar to using <<timeAllowed Parameter,the `timeAllowed` Parameter>>, when early segment termination happens values such as `numFound`, xref:faceting.adoc[Facet] counts, and result xref:stats-component.adoc[Stats] may not be accurate for the entire result set.
 
+== multiThreaded Parameter
+
+[%autowidth,frame=none]
+|===
+|Optional |Default: `false`
+|===
+
+This parameter set to `true` or `false` controls if Solr may use more than one thread to satisfy the request.
+It presently is only considered for QueryComponent to search across Lucene's segments in parallel and the xref:configuration-guide:configuring-solr-xml.adoc#indexSearcherExecutorThreads[indexSearcherExecutorThreads] value can be customised in the `solr.xml` file.
```

Comment:
"It presently is only considered" is unclear/confusing. Perhaps "Specifically this enables" 

Also: 
"segments in parallel and the xref:..." should either have a comma after parallel or be broken into two sentences (I prefer 2 sentences)

Also do we have a standardized way to mark things experimental in the docs (if not just mention it?)

Reply:
It was even more unclear before :-).  The "it presently" portion conveys it may have expanded uses later.  After all, we chose a super generic name here instead of something referencing segment parallelism.

Indeed, it'd be nice to have a standard way, maybe an icon, to label experimental stuff.  Ah well, even just textually `[EXPERIMENTAL]`